src/app/Main/Body/InfoPage/Amrs/InfoPage-AmrsDialog.component.tsx

```diff
@@ -119,7 +110,7 @@ export function InfoPageAmrDialogComponent(props: {
     );
 
     const amrTableCancelButton = t("Methods.Amrs.CancelButton");
-    const amrTableSubmitButton = ExportButtonLabelComponent(false);
+    const amrTableSubmitButton = ExportButtonLabelComponent(true);

```

Comment:
Comment: I wonder how it worked before, with the button disabled?

Reply:
It wasn't disabled. It's just a parameter for different styling.